docker-compose-dev.yml

```diff
@@ -0,0 +1,41 @@
+version: '2'
+services:
+
+  web:
```

Comment:
I'd like to align this with our foodcoops.net setup

Reply:
Ok with that.